plutus-playground-lib/src/Playground/Rollup/Render.hs

```diff
@@ -169,7 +170,10 @@ instance Render PubKey where
     render = pure . viaShow . getPubKey
 
 instance Render DataScript where
```

Comment:
Something is fishy here: we shouldn't be rendering the *data scripts*, we should be rendering the *validator scripts*. The problem is in `Render TxOutType`. Inconveniently, the `PayToScript` type only has the data script in it, the validator script hash is part of the `Address`.

We should render the data script hash as part of a script output, though.

Reply:
Unless you wildly object, I'm going to address this in a second PR. I've already started working on it but I'd like to get what we have merged.

Reply:
Okay, but it is a little important since the current thing is showing the hash of the wrong thing. Only a little important since probably nobody will notice, but still.